apps/src/templates/lessonOverview/StyledCodeBlock.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+export default class StyledCodeBlock extends Component {
+  static propTypes = {
+    programmingExpression: PropTypes.shape({
+      color: PropTypes.string,
+      name: PropTypes.string.isRequired,
+      link: PropTypes.string
+    }).isRequired
+  };
+
+  render() {
+    const {programmingExpression} = this.props;
+    console.log(programmingExpression);
```

Comment:
remove debug code?

Reply:
Its fixed in https://github.com/code-dot-org/code-dot-org/pull/39716. I just didn't merge that in here again.

Reply:
Will merge that in.

Reply:
sorry Dani, I'm fine with whatever method of chaining PRs as long as we end up removing it by the end of the chain.